test/functional/apps/dashboard/group4/dashboard_listing.ts

```diff
@@ -29,6 +29,8 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
     describe('create prompt', () => {
       it('appears when there are no dashboards', async function () {
+        await kibanaServer.savedObjects.clean({ types: ['dashboard'] });
```

Comment:
Before this was not needed, but since recently `await dashboard.initTests();` includes one dashboard, so have to clean it up to test this state 😢